bigbluebutton-html5/imports/ui/components/app/AppView.jsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import ModalContainer from '../modal/container';
 import AudioContainer from '../audio/container';
-import Tabs from '../tabs';
 import HeaderContainer from '../header';
 import FooterConatiner from '../footer';
 import Content from '../content';
+import TabsContainer from '../tabs';
```

Comment:
I think this is the new component created from scratch using tailwind, so make the folder name as first letter capital `../Tabs`

Reply:
fixed